README.md

```diff
@@ -180,10 +180,11 @@ You can find more examples [here](https://iverson.github.io/mobx-react-viewmodel
 
 ## API documentation
 
-### `useViewModel(ViewModelClass, props?)`
+### `useViewModel(ViewModelClass, props?, args?)`
 
 Basic hook that creates an instance of the `ViewModelClass` on the first render and keeps it alive during all further renders.
 If `props` is passed, it will be set to the ViewModelClass instance `props` property every time it changes.
+If `args` is passed, it will be passed to the ViewModelClass constructor ons the first render. you should give only permanent references to the object there, for example, a reference to singleton objects.
```

Comment:
A couple of typos:

"... constructor **on** the first render. **You** should **use** only ..."